src/dali_backend.cc

```diff
@@ -175,20 +177,35 @@ class DaliModelInstance : public ::triton::backend::BackendModelInstance {
     return *dali_model_;
   }
 
-  RequestMeta ProcessRequest(TRITONBACKEND_Response* response, TritonRequest& request) {
+  void Execute(std::vector<TritonRequest> requests) {
```

Comment:
Also here, couldn't this be `std::vector<TritonRequest>&`? What would be the reason for moving it?

Reply:
changed to `const std::vector<TritonRequest>&`